cosbench/cosbench.spec

```diff
@@ -0,0 +1,161 @@
+Name:          cosbench-openio
+Version:       0.4.4
+Release:       1%{?dist}
+Source0:       https://github.com/open-io/cosbench/releases/download/%{version}-openio/cosbench-%{version}-openio.tar.gz
+Source1:       cosbench-controller.service
+Source2:       controller.default.service
+Source3:       controller.log4j.properties
+
+Summary:       A benchmark tool for cloud object storage service
+License:       Apache 2.0
+Url:           https://github.com/open-io/cosbench
+BuildArch:     noarch
+
+Requires(pre): shadow-utils
+
+%{?systemd_requires}
+BuildRequires: systemd
+Requires: which
+
+%define        cosbench_home    %{_sharedstatedir}/cosbench
+%define        cosbench_prefix  %{_prefix}/cosbench
+%define        cosbench_etc     %{_sysconfdir}/cosbench
+%define        cosbench_user    cosbench
+%define        cosbench_group   %{cosbench_user}
+
+%package controller
+Summary:       Cosbench Controller
+Requires:      jre6
```

Comment:
`jre6` ? Are you sure of you ? `java-1.8.0-openjdk-headless` seems to be better

Reply:
cosbench is compiled with java1.6 and requires it at compilation: `Bundle-RequiredExecutionEnvironment: JavaSE-1.6`.

it works with newer version for sure. I don't know what's best ... real question

Reply:
@fvennetier, what do you think ?

Reply:
jre6 has been deprecated for a while. I would use the more recent version that works, and I think it is java-1.8.0-openjdk.